### PR TITLE
add gimme-acc tool for SA HMAC testing

### PR DESCRIPTION
**Pull request by jkwlui**

### CODE REVIEW

File: testing/gimme-acc/Makefile [hunk 1]
```diff
@@ -0,0 +1,17 @@
+.PHONY: bin cross upload
```
Comment (frankyn) [inline testing/gimme-acc/Makefile]: Add copyright to Makefile.

File: testing/gimme-acc/README.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+### Example use in integration tests
+
+```
+set -e -x
+
+curl https://storage.googleapis.com/gimme-acc/linux_amd64/gimme-acc > gimme-acc
+chmod +x gimme-acc
+./gimme-acc version
+
+export HMAC_SERVICE_ACCOUNT=$(./gimme-acc -project gimme-acc lease 15m)
```
Comment (frankyn) [inline testing/gimme-acc/README.md]: please switch up the project of `gimme-acc` to maybe `your-project-id`. It confused me a little bit at first seeing double.

File: testing/gimme-acc/main.go [hunk 3]
```diff
@@ -0,0 +1,288 @@
+// Copyright 2019 Google LLC
```
Comment (frankyn) [inline testing/gimme-acc/main.go]: Is there a test?

**Comment by jkwlui** [comment]
Thanks @tbpg. 

A bit of context here - we needed a service account pool to test a new feature in GCS where users are able to create keys against service accounts. Because the limitations on the number of keys you can have on a service account, languages that run parallels tests in the CI would run into problems if they used the same service accounts.

I think you're right - this is essentially the same thing as `gimmeproj`, perhaps we should just close this and reuse the same binaries but plug in a different project to `-project`? @frankyn

**Comment by frankyn** [comment]
+1, thanks @tbpg! I missed that support in this tool.

**Pull request closed without merging**